test/functional/api/v2/template_controller_test.rb

```diff
@@ -1,8 +1,30 @@
 require 'test_plugin_helper'
+require 'git'
 
 module Api
   module V2
     class TemplateControllerTest < ::ActionController::TestCase
+
+      private
+      def create_local_template_in_core(t_filename:, t_name:, model: "ProvisioningTemplate", kind: "provision", org: "", loc: "")
+        File.open("#{ForemanTemplates::Engine.root}/test/templates/core/#{t_filename}.erb", "w") { |template|
```

Comment:
Note that this creates a file in the static fixtures folder. If not used carefully, it can overwrite existing files thus breaking other tests that rely on them, so I think we should use a different path. In the ideal scenario we would have the template as a temporary file only for the duration of the test.